### PR TITLE
release 1.21.0

### PR DESCRIPTION
**Pull request by nemreid**
**Summary:** Summary of changes

## Changes

* Merge release 1.21.0 back to master

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: example/lambdas/python-reference-activity/package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "@cumulus/python-reference-activity",
   "private": true,
-  "version": "1.19.0",
+  "version": "1.21.0",
   "description": "Python reference activity",
-  "homepage": "https://github.com/nasa/cumulus/tree/master/tasks/python-reference-activity",
+  "homepage": "https://github.com/nasa/cumulus/tree/master/example/lambdas/python-reference-activity",
```
Comment (laurenfrederick) [inline example/lambdas/python-reference-activity/package.json]: Were these changes made in the release branch?

Comment (nemreid) [reply example/lambdas/python-reference-activity/package.json]: No. I had to move them to make integration tests work with the released zips as they are not built and included in the release. They are just for our testing purposes. In accordance with moving them I updated this path. There is no change to our released packages or to anything except our `example` deployment.

Comment (nemreid) [reply example/lambdas/python-reference-activity/package.json]: The two `python-reference` task packages were not published as they are set to `"private": true`.

Comment (nemreid) [reply example/lambdas/python-reference-activity/package.json]: > made in the release branch?

If you mean `release-1.21.0`, then no. If you mean `release-1.21.x` then yes.

**Pull request merged**